### ISSUE TITLE
"url" Needs to be Capitalized in Challenge

### ISSUE THREAD

**Issue by erictleung** [FreeCodeCamp/FreeCodeCamp#9116]
Challenge [Render Images from Data Sources](https://www.freecodecamp.com/challenges/render-images-from-data-sources) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

## How to Fix

[This line](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/a43b29489054f47d68fd79afeeae38456a14cde6/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L289) needs to be edited so that "url" is "URL".

## Screenshot

![image](https://cloud.githubusercontent.com/assets/2754821/15994619/a48880e4-30be-11e6-8dd2-063e867ab73e.png)

## Before Contributing

Please first read the [contributing guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before taking care of this issue. And feel free to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) chat room if you have any questions about helping. We're there to help.

**Comment by atjonathan** [comment]
@vasanthk, please submit a PR.

**Comment by vasanthk** [comment]
@atjonathan I see this is for first timers only and I agree. Hence deleted my fork. Not sure why it still shows up as a reference here.

**Comment by flipsyde606** [comment]
I am a first-timer. Can I work on this?

**Comment by vasanthk** [comment]
@flipsyde606 Yes, please do..
